spylon_kernel/scala_interpreter.py

```diff
@@ -499,12 +499,18 @@ def bind(self, name: str, value: Any, jtyp: str="Any"):
 
     @property
     def jcompleter(self):
+        """Scala code completer.
+
+        Returns
+        -------
+        scala.tools.nsc.interpreter.PresentationCompilerCompleter
```

Comment:
`scala.tools.nsc.interpreter.PresentationCompilerCompleter` is the Scala type. `py4j.java_gateway.JVMView` is a made up namespace which allows Python code to reference them I believe. There is no real Python module+type `py4j.java_gateway.JVMView. scala.tools.nsc.interpreter.PresentationCompilerCompleter`. 

What to do, what to do ...